features/original-colors/scratch-www.css

```diff
@@ -429,20 +429,9 @@ h3 span {
 .action-menu_main-button_3ccfy:hover .action-menu_main-icon_1ktMc {
     background: hsla(163, 85%, 40%, 1)
 }
-.studio-activity-icon {
-
-     filter: sepia(300%) hue-rotate(185deg) saturate(450%); 
-
+button.button {
+    background-color: var(--ste-blue) !important;
 }
```

Comment:
This is causing the search bar icon to have a blue background behind it.
<img width="509" alt="Screenshot 2023-07-11 at 5 58 51 PM" src="https://github.com/STForScratch/ScratchTools/assets/86856959/f60064b2-23a1-4ec1-9d69-4a92568dd2f9">

Reply:
> This is causing the search bar icon to have a blue background behind it. <img alt="Screenshot 2023-07-11 at 5 58 51 PM" width="509" src="https://user-images.githubusercontent.com/86856959/252826897-f60064b2-23a1-4ec1-9d69-4a92568dd2f9.png">

Alright, thanks. Just fixed it 👍